app/views/events/_form.html.slim

```diff
@@ -0,0 +1,74 @@
+= form_for(event) do |f|
+  - if event.errors.any?
+    #error_explanation
+      h2
+        = pluralize(event.errors.count, "error")
+        | prohibited this event from being saved:
+
+      ul
+        - event.errors.full_messages.each do |message|
+          li= message
+
+  .form-inline
+    .form-group
+      = f.label :'Event Name: '
+      = f.text_area :event_name, class: "form-control", placeholder: "name of the event"
+    ' 
+    .form-group
+      = f.label :'Location: '
+      ' 
+      = f.text_area :location, class: "form-control", placeholder: "location of the event (including postal code)"
+    ' 
+    .form-group
+      = f.label :'Number of Volunteers: '
+      ' 
+      = f.text_area :number_of_volunteers, type: "number", class: "form-control", placeholder: "total number of volunteers required"
```

Comment:
the `type: 'number'` here isn't working. you can use `f.number_field`. Refer to [this](http://apidock.com/rails/ActionView/Helpers/FormHelper) for more info on what fields you can use (check out the links at the right side)

@G115H and @lingadingdong take note of this too